app/upgrades/v2_5_0/v2_5_0.go

```diff
@@ -53,145 +51,44 @@ var Upgrade = upgrades.Upgrade{
 	StoreUpgrades: storetypes.StoreUpgrades{},
 }
 
-// Set of addresses that held stNIBI in ERC20 form prior to the v2.5.0 upgrade
-func MAINNET_STNIBI_HOLDERS() []gethcommon.Address {
```

Comment:
In my strategy, we actually don't even need to keep track of the original stNIBI holders. It comes from the clever realization that the old and new ERC20 contracts have the exact same storage slot layouts, so as long as we don't mess with the balances mapping in the old ERC20 contract, everybody's balances should automatically be maintained. No need to track everybody's balances in memory.